src/components/KSegmentedControl/KSegmentedControl.vue

```diff
@@ -34,6 +34,37 @@ export interface SegmentedControlOption {
   disabled?: boolean
 }
 
+// functions used in prop validators
+const getValues = (items: SegmentedControlOption[]) => {
+  const vals:string[] = []
+  items.forEach((item: SegmentedControlOption) => vals.push(item.value + ''))
```

Comment:
question: are you appending an empty string to ensure `item.value` is a string?

What if `item.value` is `null` or `undefined`? Do we still want to store an empty string?

Reply:
`value` is required, so it should fail the validator in that scenario.